1122.py

```diff
@@ -3,8 +3,10 @@
 
 class Solution:
     def relativeSortArray(self, arr1: List[int], arr2: List[int]) -> List[int]:
-        only_arr1 = sorted([e for e in arr1 if e not in arr2])
-        num_pos = defaultdict(lambda: 0)
+        arr1_set = set(arr1)
```

Comment:
`arr1_set` is not needed. There isn't any **look-up** operation related to elements in `arr1` in the rest of the program. In line 8, you will go through each element in `arr1` anyway.